packages/subiquity_client/lib/src/types.dart

```diff
@@ -255,6 +265,8 @@ class Disk with _$Disk {
     String? vendor,
   }) = _Disk;
 
+  List<Partition>? get partitions => objects?.whereType<Partition>().toList();
```

Comment:
Note: The `partitions` list from subiquity includes now both, `Partition` and `Gap` objects, which breaks the UI and a lot of tests. Renaming the mixed object list as `objects` and providing `partitions` for backward compatibility reduces the amount of diff a lot.

Reply:
That's very considerate, thanks!